packages/connect/src/types/device.ts

```diff
@@ -56,7 +57,7 @@ export type FirmwareRevisionCheckResult =
       };
 
 type BaseDevice = {
-    path: string;
```

Comment:
this is the change - type tightening of device.path. Till now, it was only internally defined as PathPublic (branded type). From now on, this branded type is propagated to higher layers.